web/src/views/DandisetLandingView/DandisetPublish.vue

```diff
@@ -370,6 +422,15 @@ import { draftVersion, VALIDATION_ICONS } from '@/utils/constants';
 import { RawLocation } from 'vue-router';
 import { open as openMeditor } from '@/components/Meditor/state';
 
+const PUBLISH_CHECKLIST = [
+  'a descriptive title (e.g., Data related to foraging behavior in bees vs Smith et al 2022)',
```

Comment:
I don't think "Data related to foraging behavior in bees vs Smith et al 2022" is a good example of a dandiset title (specifically because of the "vs Smith et al 2022" part).

Ah--the "vs" is introducing an example of a bad title? (@satra, do I have that right?) If so, then we should change this to:

```suggestion
  'a descriptive title (e.g., "Data related to foraging behavior in bees" rather than "Smith et al 2022")',
```

(This is also a case where I would prefer to use italics to highlight the text, which relates to my question above about HTML content inside of a `<v-list-item>`.)

Reply:
> (This is also a case where I would prefer to use italics to highlight the text, which relates to my question above about HTML content inside of a `<v-list-item>`.)

https://github.com/dandi/dandi-archive/pull/1122/commits/444804d5e5155b623938f45eb9d3b20a6f04c34f